### PR TITLE
Che 684 improve import from SSH error notification

### PR DESCRIPTION
**Pull request by olexii4**
**_2 Upvotes**_ @ashumilova WDYT?

### CODE REVIEW

**Comment by olexii4** [comment]
selection_061

File: dashboard/src/app/projects/create-project/add-ssh-key/add-ssh-key.controller.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  /**
+   * Update config url.
+   */
+  updateConfigURL() {
+    let workspaces = this.cheAPI.getWorkspace().getWorkspaces();
+
+    let ctrl = this;
+    let findWorkspace = this.lodash.find(workspaces, function (workspace) {
```
Comment (benoitf) [inline dashboard/src/app/projects/create-project/add-ssh-key/add-ssh-key.controller.js]: with arrow function, no need to do 'let ctrl = this" we can use inside function return workspace.id === this.workspaceId

**Comment by olexii4** [comment]
DONE

**Comment by ashumilova** [comment]
+1

**Comment by benoitf** [comment]
+1 with little fixes

**Pull request merged**